x11-packages/fcitx5-hangul/build.sh

```diff
@@ -4,7 +4,7 @@ TERMUX_PKG_LICENSE="LGPL-2.1"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="5.1.3"
 TERMUX_PKG_SRCURL="https://github.com/fcitx/fcitx5-hangul/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz"
-TERMUX_PKG_SHA256=44fa27b0d0168f9e41d24da3e0f3818e09c15c11f2f99261d13134bcef478e45v
```

Comment:
How did that even pass the linter with a `v` at the of the SHA?
https://github.com/termux/termux-packages/blob/887e52732efe5c3d7470c38f798d8414f71fd135/scripts/lint-packages.sh#L329-L336